src/manual/task-execution/provisioning.md

```diff
@@ -0,0 +1,46 @@
+---
+filename: task-execution/provisioning.md
+title: Provisioning
+order: 40
+---
+
+In the simplest case, a worker implementation is installed and configured
+manually on a host and executes tasks indefinitely. For most purposes, it is
+far more cost-effective to dynamically adjust the pool of available workers
+based on the work to be done. This process is called provisioning.
+
+A provisioner operates by monitoring the pool of running workers and the
+corresponding queues. Based on the queue length and the configuration of the
+provisioner, it may create new workers or signal to running workers that they
+should stop. Of course, provisioners might take into account additional
+information to make better predictions of future load. A provisioner's
+configuration can balance performance (many instances to execute tasks in
+parallel) against cost (from idle workers).
+
+## Provisioner IDs
+
+Worker types are scoped within a `provisionerId`, allowing each provisioner its
+own "namespace" of managed workers.
+
+## AWS Provisioner
+
+The Taskcluster team provides a provisioner instance, running the AWS
+Provisioner, which creates EC2 spot instances within the Taskcluster AWS
+account.
+
+It is identified as provisionerId `aws-provisioner-v1`. Its administrative
+interface is available at https://tools.taskcluster.net/aws-provisioner/.  This
+interface allows monitoring of current load, as well as management of worker
+types.
+
+The worker types provided by the AWS provisioner are a mix: some are
+Firefox-specific, while others are available for other Mozilla uses. Since the
+service runs with the Taskcluster team's AWS credentials, all costs are borne
+by the Taskcluster team.
+
+If you are setting up a new worker type with a new AMI, it is your
+responsibility to ensure that
```

Comment:
Are you suggesting adding "under the AWS provisioner" or omitting "with a new AMI"?  This is under the "AWS Provisioner" heading, so I thin the first is implied. And if you're setting up a new worker type without a new AMI, then you don't need to do anything to ensure that..